packages/integrations/clearinghouse/src/parsers/x12-271-parser.ts

```
@@ -482,4 +487,338 @@ export class X12_271_Parser {
+
+    // Financial terms based on EB05 (Plan Coverage Description)
+    if (eb05_planCoverage && eb07_value) {
+      const monetaryValue = parseFloat(eb07_value);
+      if (!isNaN(monetaryValue)) {
+        switch (eb05_planCoverage) {
+          case 'HM': // Copayment
+            benefit.copayCents = this.dollarsToCents(monetaryValue);
+            break;
+          case 'HN': // Deductible
+            // EB08 might contain amount met
+            const deductibleMet = eb08_value ? parseFloat(eb08_value) : undefined;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)